cl/stats/metrics.py

```diff
@@ -0,0 +1,31 @@
+from collections.abc import Callable
+
+from prometheus_client import Counter
+
+search_queries_total = Counter(
+    "cl_search_queries_total",
+    "Total number of search queries",
+    ["query_type", "method"],
+)
+
+
+def _inc(metric: Counter, inc: int, **kwargs):
+    metric.labels(**kwargs).inc(inc)
+
+
+def prometheus_handler(key, *args):
```

Comment:
This function name should be a verb:
 - log_to_prometheus?
 - prom_log?
 - tally_stat_prometheus?
 - tally_in_prometheus?
 - tally_to_prometheus?
 - tally_prometheus?

That'll make it more clear that it does when you call it elsewhere in your code.